src/main/java/org/broadinstitute/dsm/route/PatchRoute.java

```diff
@@ -263,6 +275,18 @@ else if (participantDataId != null) {
                         }
                         return new Result(200, new GsonBuilder().serializeNulls().create().toJson(map));
                     }
+                    else if (Patch.DDP_PARTICIPANT_ID.equals(patch.getParent())) {
+                        //new additional value for pt which is not in ddp_participant and ddp_participant_record table yet
+                        DDPInstance ddpInstance = DDPInstance.getDDPInstance(patch.getRealm());
+                        int participantId = insertDdpParticipant(patch, ddpInstance);
+                        insertDdpParticipantRecord(participantId);
+                        if (participantId > 0) {
+                            DBElement dbElement = patchUtil.getColumnNameMap().get(patch.getNameValue().getName());
+                            if (dbElement == null) throw new RuntimeException("DBElement not found in ColumnNameMap: " + patch.getNameValue().getName());
+                            Patch.patch(String.valueOf(participantId), patch.getUser(), patch.getNameValue(), dbElement);
+                            return new Result(200, new GsonBuilder().serializeNulls().create().toJson(Map.of(PARTICIPANT_ID, String.valueOf(participantId))));
```

Comment:
because of lack of time decided to keep `Result` since `PatchRoute` is used widely on frontend side and frontend code uses `result.code`, `result.body` and changing it here would break lots of things on frontend